### PR TITLE
Add Client SDK Deprecation Policy

### PR DESCRIPTION
**Pull request by scannillo**
### Summary of changes

 - Add Client SDK Deprecation policy link & chart to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,6 +248,15 @@ Take a look at `BTCardFormViewController.m` to see examples of using the form fi
 
 Start with [**'Hello, Client!'**](https://developers.braintreepayments.com/ios/start/hello-client) for instructions on basic setup and usage.
 
+## Versions
+
+This SDK abides by our Client SDK Deprecation Policy. For more information on the potential statuses of an SDK check our [developer docs](http://developers.braintreepayments.com/guides/client-sdk/deprecation-policy).
```
Comment (hollabaq86) [inline README.md:253]: abides

Comment (hollabaq86) [reply README.md:253]:
```suggestion
This SDK abides by our Client SDK Deprecation Policy. For more information on the statuses of an SDK, check our [developer docs](http://developers.braintreepayments.com/guides/client-sdk/deprecation-policy).[Minimum supported versions](https://developers.braintreepayments.com/guides/client-sdk/setup/ios/#requirements) are also available in our developer docs.
```

Comment (hollabaq86) [reply README.md:253]: One note about my suggested changes - these link to the minimum supported version of the Core SDK. I'm wondering if we should use the link to Drop-In's [setup page](https://developers.braintreepayments.com/guides/drop-in/setup-and-integration/ios/v5#setup) instead. Only think if the Drop-In setup page doesn't mention minimum supported versions... maybe we should update that.

**Review by hollabaq86 — COMMENTED** [review]
Some suggested tweaks to the language to bring it inline with the server SDK blurbs about deprecation policies.

One other general note - the README for this library is getting pretty large, what do folks think about adding a Table of Contents with links to the top?

File: README.md (hunk 1 above)
Comment (scannillo) [reply README.md:253]: Good call - I added a JIRA (#DTBTSDK-720) to update the Drop-In Developer Docs with the minimum supported versions. 

Once we do - we can make this change to the README. How does that sound, and we add that in a later PR?

File: README.md [hunk 2]
```diff
@@ -248,6 +248,15 @@ Take a look at `BTCardFormViewController.m` to see examples of using the form fi
...
+
+| Major version number | Status | Released | Deprecated | Unsupported |
+| -------------------- | ------ | -------- | ---------- | ----------- |
+| 9.x.x | Beta | March 2021 | TBA | TBA |
```
Comment (sestevens) [inline README.md:257]:
```suggestion
| 9.x.x | Beta | TBA | TBA | TBA |
```

Should we just put TBA for the v9 release date? Or omit it for now and add it when we actually do a beta release?

**Comment by sestevens** [comment]
+1 to @hollabaq86's idea of adding a table of contents to the README.

File: README.md [hunk 3]
```diff
@@ -12,6 +12,14 @@ Welcome to Braintree's Drop-In SDK for iOS!
 
 **The Braintree iOS Drop-In SDK requires Xcode 12+**. It permits a deployment target of iOS 12.0 or higher.
 
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+1. [Documentation](#documentation)
+1. [Demo App](#demo)
+1. [Help](#help)
+1. [Feedback](#feedback)
```
Comment (sestevens) [inline README.md:21]: Should we add the Versions and License sections here?

**Review by sestevens — APPROVED** [review]
Looks good to me! Good call adding the TODO for v9. That would be easy to forget.

**Pull request merged**